wavelength-plugins/exp.js

```diff
@@ -220,44 +227,47 @@ exports.commands = {
 		let parts = target.split(',');
 		let targetUser = parts[0].toLowerCase().trim();
 		if (!this.can('exp')) return false;
-		if (cmd !== 'confirmresetexp') {
-			return this.popupReply('|html|<center><button name="send" value="/confirmresetexp ' + targetUser + '"style="background-color:red;height:300px;width:150px"><b><font color="white" size=3>Confirm XP reset of ' + WL.nameColor(targetUser, true) + '; this is only to be used in emergencies, cannot be undone!</font></b></button>');
+		if (cmd !== "confirmresetexp") {
+			return this.popupReply(`|html|<center><button name="send" value="/confirmresetexp ${targetUser}" style="background-color: red; height: 300px; width: 150px"><strong><font color= "white" size= 3>Confirm XP reset of ${WL.nameColor(targetUser, true)}; this is only to be used in emergencies, cannot be undone!</font></strong></button>`);
 		}
 		Db.exp.set(toId(target), 0);
 		if (Users.get(target)) Users.get(target).popup('Your XP was reset by an Administrator. This cannot be undone and nobody below the rank of Administrator can assist you or answer questions about this.');
-		user.popup("|html|You have reset the XP of " + WL.nameColor(targetUser, true) + ".");
-		Monitor.adminlog('[EXP Monitor] ' + user.name + ' has reset the XP of ' + target);
+		user.popup(`|html|You have reset the XP of ${WL.nameColor(targetUser, true)}.`);
+		Monitor.adminlog(`[EXP Monitor] ${user.name} has reset the XP of ${target}`);
 		room.update();
 	},
 
-	doublexp: 'doubleexp',
+	doublexp: "doubleexp",
```

Comment:
why is this ``"`` and the others are ``'``?

Reply:
Looks cleaner imo

Reply:
my question is... why are the other alias' using ``'`` when this one is the only one using ``"``